website/content/docs/connect/cluster-peering/create-manage-peering.mdx

```diff
@@ -0,0 +1,137 @@
+---
+layout: docs
+page_title: Create and Manage Peering Connections
+description: >-
+    This page describes how to use the Consul CLI to create, manage, and delete peering connections for cluster peering.
+---
+
+# Create and Manage Peering Connections
+
+~> This page covers features that are currently in _technical preview_. Features and functionality are subject to change. You should never use the technical preview release in secure environments or production scenarios. Features in technical preview may face performance and scaling issues, with limited support options available.
+
+A peering token enables cluster peering between different datacenters. Once you generate a peering token, you can use it to establish a connection between clusters. Then you can export services and authorize other clusters to call those services.
+
+To peer clusters, you must complete the following steps in order:
+
+1. Create a peering token
+1. Establish a connection between clusters
+1. Export service endpoints
+1. Authorize connections between peers
+
+## Create a peering token
+
+You can generate peering tokens and initiate connections from either the server agents or the client agents in your clusters. For the current release, we recommend you initiate peering through the client agents in the partitions you want to connect.
+
+To begin the cluster peering process, generate a peering token in one of your clusters. The other cluster uses this token to establish the peering connection.
+
+In “cluster-01,” issue a request for a peering token using the HTTP API.
+
+```shell-session
+$ curl -X POST --data '{"PeerName":"cluster-02"}' http://localhost:8500/v1/peering/token
```

Comment:
nit: use the long-form of CLI flags so that they're more easily readable.
```suggestion
$ curl --request POST --data '{"PeerName":"cluster-02"}' --url http://localhost:8500/v1/peering/token
```